backend/app/adapter/db/userurl_test.go

```diff
@@ -30,29 +58,19 @@ func TestNewUserURLRelationSQL(t *testing.T) {
 			mdtest.Equal(t, nil, err)
 			defer db.Close()
 
-			statement := fmt.Sprintf(
-				`INSERT INTO "%s" (.+) VALUES (.+)`,
-				table.UserURLRelation.TableName,
-			)
+			expQuery := fmt.Sprintf(`^SELECT "%s" FROM "%s" WHERE "%s"=.+$`, table.UserURLRelation.ColumnURLAlias,
+				table.UserURLRelation.TableName, table.UserURLRelation.ColumnUserEmail)
+			stub.ExpectQuery(expQuery).WillReturnRows(testCase.tableRows)
 
-			if testCase.expHasErr {
-				stub.ExpectExec(statement).WillReturnError(errors.New("row exists"))
-			} else {
-				stub.ExpectExec(statement).WillReturnResult(driver.ResultNoRows)
-			}
-
-			userURLRelationRepo := NewUserURLRelationSQL(db)
-
-			url := entity.URL{
-				Alias: testCase.alias,
-			}
-			err = userURLRelationRepo.CreateRelation(testCase.user, url)
+			userURLRelation := NewUserURLRelationSQL(db)
+			result, err := userURLRelation.FindAliasesByUser(mockedUser)
```

Comment:
```suggestion
			result, err := userURLRelationRepo.FindAliasesByUser(testCase.user)
```

Reply:
We are examining a specific test case here. Hence, we should use the input provided by the test case.